src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.skia.cs

```diff
@@ -21,11 +21,12 @@ internal class TextBoxView
 
 		public TextBoxView(TextBox textBox)
 		{
+			_textBox = new WeakReference<TextBox>(textBox);
```

Comment:
I failed to notice this one before, but we should be using the WeakReferencePool here. All DependencyObject instance have a pre-build weak ref, which avoid creating too many of those.

Reply:
@jeromelaban Does it need to be a WeakReference at all? There is no iOS circularity involved at all as this code is just Skia.

Reply:
Hmm, actually, TextBoxView's lifetime matches TextBox, so there is no reason to make the reference weak.

Reply:
Yes, with the new pooling that's coming, it will be required